fix: update rules (#199)
SLNE-Development/community-server-docs: Writerside/topics/rules/rules.md

   tatsächliche Absicht zu haben, aktiv am Spielgeschehen teilzunehmen. Sollte auffallen, dass Spieler und/oder deren
   Freunde diese Mechaniken ausnutzen, wird dies mit einem Ausschluss für beide geahndet.

> 6. **Die Verbreitung unangemessener oder <tooltip term="harmful-content">jugendgefährdende Inhalte</tooltip> ist strengstens untersagt.**

Reviewer: ```suggestion
6. **Die Verbreitung unangemessener oder <tooltip term="harmful-content">jugendgefährdender Inhalte</tooltip> ist strengstens untersagt.**
```

   Es ist strikt verboten, beleidigende, diskriminierende, rassistische,

>    sexistische, antisemitistische, extremistische, gewaltverherrlichende, rechtswiedrige oder anderweitig unangemessene Inhalte in jeglicher Form zu äußern, darzustellen, zu verbreiten oder zu bewerben.

Reviewer: ```suggestion
   sexistische, antisemitistische, extremistische, gewaltverherrlichende, rechtswidrige oder anderweitig unangemessene Inhalte in jeglicher Form zu äußern, darzustellen, zu verbreiten oder zu bewerben.
```

   Dies betrifft unter anderem den Chat, Bücher, Schilder, Item-Namen, Skins, Bauwerke und jegliche andere Darstellungen und Ausdrucksformen.

>    > Casinos werden aufgrund des Bestandschutzes auf dem Survival Server aktuell noch geduldet. In der 1.22 werden diese jedoch auch dort untersagt sein.

Reviewer 2: "In der 1.22 werden diese jedoch auch dort untersagt sein." - nicht lieber "auf dem nächstem Server" oder "nächste Session"?

Author: https://github.com/SLNE-Development/community-server-docs/pull/199/commits/f66a75a9aba8c6853dd6c8b5af182b20c3377f5e

>    > Casinos werden aufgrund des Bestandschutzes auf dem Survival Server aktuell noch geduldet. Auf dem nächsten Server werden diese jedoch untersagt sein.

Reviewer: ```suggestion
   > Casinos werden aufgrund des Bestandschutzes auf dem Survival-Server aktuell noch geduldet. Auf dem nächsten Server werden diese jedoch untersagt sein.
```

   >
   {style="note"}   

>    > - Als Scherz oder Ironie gemeinte Aussagen werden nicht toleriert und können Konsequenzen mit sich ziehen.

Reviewer 3: nach sich ziehen?

Author: https://github.com/SLNE-Development/community-server-docs/pull/199/commits/17cdee5827f7a32132e550b24d31798afdc7196c

   > - Das Zitieren oder Nachahmen unangemessener Aussagen, unabhängig vom Kontext, wird nicht toleriert.
   > - Das bewusste Ansprechen oder Nachfragen nach verbotenen oder unangemessenen Inhalten wird ebenfalls geahndet.

>    > - Falls ihr Zeuge eines Regelverstoßes werdet, meldet dies bitte umgehend im [Support](support.md) bei der Moderatrion Moderator und stellt, wenn möglich, Beweise (z.B. Screenshots, Chatlogs, Videos) bereit.

Reviewer 3: ?

Author: https://github.com/SLNE-Development/community-server-docs/pull/199/commits/8fb3c60c1d613cb619e9217d51ce01085f464a9e

   >

>    {style="warning" title="Ein Verstoß kann zu einem permanenten Ausschluss vom gesamten Netzwerk und allen anderen Plattformen führen!"}

Author: Man kann keinen tooltip in den titel packen

>    > - Falls ihr Zeuge eines Regelverstoßes werdet, meldet dies bitte umgehend im [Support](support.md) bei der Moderation und stellt, wenn möglich, Beweise (z.B. Screenshots, Chatlogs, Videos) bereit.

Reviewer: ```suggestion
   > - Falls ihr Zeuge eines Regelverstoßes werdet, meldet dies bitte umgehend im [Support](support.md) bei der Moderation und stellt, wenn möglich, Beweise (z. B. Screenshots, Chatlogs, Videos) bereit.
```

7. **Moderatoren und Admins haben stets das letzte Wort**, auch wenn dies im Widerspruch zu den offiziellen Regeln
   stehen sollte. Wir bitten darum, auf längere Diskussionen zu verzichten. Wenn euch ein Moderator oder Admin
   auffordert, etwas zu unterlassen, respektiert dies bitte ohne weitere Erklärungen. Es gibt immer einen guten Grund
   für solche Entscheidungen.

> Sobald du den Server betrittst, bestätigst du automatisch, dass du

> > das Regelwerk gelesen und akzeptiert hast.\

Reviewer: ```suggestion
> das Regelwerk gelesen und akzeptiert hast.
```

   Advent calendar, and then immediately disconnect without a genuine intent to actively participate in the gameplay. If
   it is discovered that players and/or their friends are exploiting such mechanics, both will face a ban.

6. **The dissemination of inappropriate or <tooltip term='harmful-content$English'>content harmful to minors</tooltip> is strictly prohibited.**

>    It is strictly forbidden to disseminate offensive, discriminatory, racist, sexist, anti-Semitic
>    sexist, extremist, violence-glorifying, illegal or otherwise inappropriate content in any form.
>    This applies, among other things, to chat, books, signs, item names, skins, buildings and any other representations and forms of expression.
>    

Reviewer: ```suggestion
   It is strictly forbidden to express, depict, distribute, or promote offensive, discriminatory, racist, sexist, anti-Semitic, extremist, violence-glorifying, illegal, or otherwise inappropriate content in any form.
This applies, among other things, to chat, books, signs, item names, skins, buildings, and any other representations and forms of expression.
   
```

   > Casinos are currently tolerated on the Survival Server due to legacy protection. However, in version 1.22 they will also be prohibited there.
   >
   {style="note"}

>    > - Statements made as jokes or irony will not be tolerated and may lead to consequences.

Reviewer: ```suggestion
   > - Statements intended as jokes or irony will not be tolerated and may result in consequences.
```

>    > - Quoting or imitating inappropriate statements, regardless of context, will not be tolerated.

Reviewer: ```suggestion
   > - Quoting or mimicking inappropriate statements, regardless of context, is strictly prohibited.
```

>    > - Deliberately addressing or inquiring about prohibited or inappropriate content will also be penalized.

Reviewer: ```suggestion
   > - Deliberately bringing up or inquiring about prohibited or inappropriate content will also be penalized.
```

   > - If you witness a rule violation, please report it immediately via [support](support.md) to the server staff and provide evidence (e.g., screenshots, chat logs, videos) if possible.
   >

>    {style="warning" title="A violation may result in a permanent ban from the entire network and all other platforms!"}

Reviewer 3: Selbiges hier wie bei deutsch

7. **Moderators and admins have the final say**, even if it contradicts the official rules. We kindly ask that you
   refrain from engaging in lengthy discussions. If a moderator or admin asks you to stop doing something, please
   respect their request without further argument. There is always a valid reason for such decisions.

> > As soon as you enter the server, you automatically confirm that you have read and accepted the rules.\
> > Ignorance is no defence against punishment.

Reviewer: ```suggestion
> By entering the server, you automatically confirm that you have read and accepted the rules.
> Ignorance is no excuse for violations.
```
